templates/repo/issue/labels/label_list.tmpl

```diff
@@ -1,6 +1,14 @@
-<h4 class="ui top attached header">
-	{{.locale.Tr "repo.issues.label_count" .NumLabels}}
-	<div class="ui right">
+<h4 class="ui top attached header gt-df">
+	<span class="gt-f1">{{.locale.Tr "repo.issues.label_count" .NumLabels}}</span>
+	<label class="gt-f1" data-tooltip-content={{.locale.Tr "repo.issues.label_archive_tooltip"}}>
```

Comment:
The position seems pretty strange. 

IMO it should be either left aligned or right aligned.

![image](https://github.com/go-gitea/gitea/assets/2114189/8927a1b7-787c-4cb9-bed5-665a41e704e7)